samples/basic/clock_pwm/dts/bindings/zephyr,user.yaml

```diff
@@ -0,0 +1,12 @@
+description: |
+  Example binding for a node using a clock from a pwm
+  source.
+
+compatible: "zephyr,user"
```

Comment:
This is not how the zephyr,user compatible is supposed to be used.

Reply:
thanks, I'll have a look in more detail.